docs/config.md

```diff
@@ -251,6 +251,10 @@ pipeline:
 validation:
   microDeposits:
     [ sameyDay: <boolean> ]
+    withdraw:
+      # Regulation D from the Federal Reserve limits withdraws from savings accounts
+      # to six per month with fees for extra withdraws. This has been temporarily suspended in 2020.
+      [ fromSavingsAccounts: <boolean> ]
```

Comment:
@adamdecaf what do you think about renaming this option into something like:
`skipForSavingsAccounts`? Right now I read it that system withdraws from saving accounts only (if true).

Reply:
When I wrote this I was thinking the default should be safer (to not withdraw). In my head the config read as "withdraw from savings accounts" = false. 

If the default is "don't skip" I worry users of PayGate can incur fees accidentally for their end-users. 

Thanks for checking out this PR btw!